fixture-monkey-kotlin/src/test/kotlin/com/navercorp/fixturemonkey/kotlin/KotlinInnerSpecTest.kt

```diff
@@ -0,0 +1,915 @@
+/*
+ * Fixture Monkey
+ *
+ * Copyright (c) 2021-present NAVER Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.navercorp.fixturemonkey.FixtureMonkey
+import com.navercorp.fixturemonkey.api.generator.ArbitraryContainerInfo
+import com.navercorp.fixturemonkey.api.type.TypeReference
+import com.navercorp.fixturemonkey.customizer.Values
+import com.navercorp.fixturemonkey.customizer.Values.NOT_NULL
+import com.navercorp.fixturemonkey.kotlin.KotlinPlugin
+import com.navercorp.fixturemonkey.kotlin.giveMeBuilder
+import com.navercorp.fixturemonkey.kotlin.kotlinInnerSpec
+import com.navercorp.fixturemonkey.kotlin.setKotlinInner
+import com.navercorp.fixturemonkey.kotlin.InnerSpecTestSpecs.ComplexObject
+import com.navercorp.fixturemonkey.kotlin.InnerSpecTestSpecs.ComplexObjectObject
+import com.navercorp.fixturemonkey.kotlin.InnerSpecTestSpecs.IntegerMapObject
+import com.navercorp.fixturemonkey.kotlin.InnerSpecTestSpecs.ListStringObject
+import com.navercorp.fixturemonkey.kotlin.InnerSpecTestSpecs.MapObject
+import com.navercorp.fixturemonkey.kotlin.InnerSpecTestSpecs.NestedKeyMapObject
+import com.navercorp.fixturemonkey.kotlin.InnerSpecTestSpecs.NestedListStringObject
+import com.navercorp.fixturemonkey.kotlin.InnerSpecTestSpecs.SimpleObject
+import com.navercorp.fixturemonkey.kotlin.InnerSpecTestSpecs.SupplierStringObject
+
+import net.jqwik.api.Arbitraries
+import net.jqwik.api.Property
+import org.assertj.core.api.BDDAssertions.then
+import org.assertj.core.api.BDDAssertions.thenThrownBy
+import java.util.function.Supplier
+import java.util.stream.Collectors
+
+class KotlinInnerSpecTest {
```

Comment:
Could you please move it to the 'fixture-monkey-tests/kotlin-test' module?

Reply:
The review comments have been incorporated in [commit](https://github.com/naver/fixture-monkey/pull/1196/commits/c5093b541c0fd44f089a81dab73c4b659182a046)